### PR TITLE
Add Indonesian translation

### PR DESCRIPTION
**Pull request by mochamadboval**
This PR adds file `id.toml` that includes Indonesian translation.

### CODE REVIEW

File: i18n/id.toml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+[selectLanguage]
+other = "Pilih Bahasa"
+
+[switchTheme]
+other = "Ganti Tema"
+# === partials/header.html ===
+
+# === partials/footer.html ===
+[poweredBySome]
+other = "Ditenagai oleh {{ .Hugo }} | Tema - {{ .Theme }}"
```
Comment (khusika) [inline i18n/id.toml:49]: `Didukung oleh....`

File: i18n/id.toml [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+# === partials/footer.html ===
+
+# === partials/comment.html ===
+[valineLang]
+other = "en"
+
+[valinePlaceholder]
+other = "Komentarmu ..."
```
Comment (khusika) [inline i18n/id.toml:57]: `Komentar anda ...`

File: i18n/id.toml [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+
+[facebookLanguageCode]
+other = "id"
```
Comment (khusika) [inline i18n/id.toml:60]: I'm not sure about this, but based on [Facebook internationalization](https://developers.facebook.com/docs/internationalization/) should be `id_ID`

File: i18n/id.toml [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+# === partials/comment.html ===
+
+# === partials/assets.html ===
+[search]
+other = "Pencarian"
```
Comment (khusika) [inline i18n/id.toml:65]: Search without _-ing_ mean `Cari`

File: i18n/id.toml [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+other = "dilihat"
+
+[author]
+other = "Penulis"
+
+[updatedOnDate]
+other = "Diperbarui pada {{ .Date }}"
+
+[readMarkdown]
+other = "Lihat versi Markdown"
```
Comment (khusika) [inline i18n/id.toml:128]: I think it is better to translate it into `Baca Markdown`

File: i18n/id.toml [hunk 6]
```diff
@@ -0,0 +1,195 @@
...
+
+# === shortcodes/version.html ===
+[new]
+other = "BARU"
+
+[changed]
+other = "BERUBAH"
+
+[deleted]
+other = "DIHAPUS"
```
Comment (khusika) [inline i18n/id.toml:194]: _deleted_ is a past tense, so it should be `TERHAPUS`

File: i18n/id.toml [hunk 7]
```diff
@@ -0,0 +1,195 @@
...
+other = "Sukses"
+
+[question]
+other = "Pertanyaan"
+
+[warning]
+other = "Peringatan"
+
+[failure]
+other = "Kegagalan"
```
Comment (khusika) [inline i18n/id.toml:171]: `Gagal`

File: i18n/id.toml [hunk 8]
```diff
@@ -0,0 +1,195 @@
...
+other = "Hasil tidak ditemukan"
+
+[lunrLanguageCode]
+other = "en"
+
+[copyToClipboard]
+other = "Salin ke papan klip"
+
+[cookieconsentMessage]
+other = "Situs ini menggunakan Cookies untuk meningkatkan pengalaman Anda."
```
Comment (khusika) [inline i18n/id.toml:86]: `...anda.` not `...Anda.`

File: i18n/id.toml (hunk 1 above)
Comment (mochamadboval) [reply i18n/id.toml:49]: IMO, "Didukung oleh" is more precisely translated as "Supported by". I remember WordPress also translated "Powered by" to "Ditenagai oleh".

File: i18n/id.toml (hunk 2 above)
Comment (mochamadboval) [reply i18n/id.toml:57]: Ahh, this seems more formal, thank you.

File: i18n/id.toml (hunk 3 above)
Comment (mochamadboval) [reply i18n/id.toml:60]: I checked again and you're right.

File: i18n/id.toml (hunk 5 above)
Comment (mochamadboval) [reply i18n/id.toml:128]: I translated it to "Lihat versi Markdown" to clarify the purpose of the menu, but you're right and should be translated to "Baca Markdown".

File: i18n/id.toml (hunk 8 above)
Comment (mochamadboval) [reply i18n/id.toml:86]: In KBBI, word "Anda" should be capitalize.

Thank you for reviewing my PR. This is my first time contributing to someone else project. Once again, thank you :)

File: i18n/id.toml (hunk 1 above)
Comment (khusika) [reply i18n/id.toml:49]: > IMO, "Didukung oleh" is more precisely translated as "Supported by". I remember WordPress also translated "Powered by" to "Ditenagai oleh".

1. As far is i know, there are no references to use conjuction with **_di-i_**, so _Didukung_ is a wrong word.
2. The translation is come from **_Powered_** followed with _**by**_, so the exact translation is _**Didukung oleh**_ or _**Dipersembahkan oleh**_

Comment (mochamadboval) [reply i18n/id.toml:49]: **Dipersembahkan oleh** seems good for this translation, but I found another option, **Diberdayakan oleh**. What do you think?

**Review by khusika — APPROVED** [review]

**Comment by mufidu** [comment]
Awesome job, guys. Thanks.

File: i18n/id.toml (hunk 6 above)
Comment (mufidu) [reply i18n/id.toml:194]: Or maybe `TELAH DIHAPUS`

File: i18n/id.toml [hunk 9]
```diff
@@ -0,0 +1,195 @@
...
+[quote]
+other = "Kutipan"
+# === shortcodes/admonition.html ===
+
+# === shortcodes/version.html ===
+[new]
+other = "BARU"
+
+[changed]
+other = "BERUBAH"
```
Comment (mufidu) [inline i18n/id.toml:191]: I think `TELAH DIUBAH` is more fit in this context.

**Comment by owl4ce** [comment]
Previously I was about to open a new PR, but in the end I saw this. [Here's my personal version of the translation](https://github.com/owl4ce/owl4ce.github.io/blob/loveit/i18n/id.toml).

**Pull request merged**